SITnett/SITnett/views.py

```diff
@@ -12,22 +12,20 @@
 
 
 def get_ar(arstall: int) -> models.Ar:
-    """Finner et gitt år, eller oppretter det hvis det ikke ligger inne i databasen."""
+    # finner et gitt år, eller oppretter det hvis det ikke ligger inne i databasen.
```

Comment:
Bakgrunnen for å endre fra `#` til `"""` er at sistnevnte tolkes som dokumentasjon av funksjonen, og i de fleste editorer dukker opp som en forklarende tekst når du holder over funksjonsnavnet.
<img width="470" alt="image" src="https://github.com/user-attachments/assets/3adba853-1500-407d-9db0-b234ec468008" />

Reply:
Okei, skjønner! Det hørtes jo fint ut, men da kan jeg legge inn en egen PR der vi endrer til """ for alle funksjonene.